packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -278,7 +274,10 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
             this.realizeCore(this.existing).catch((error) => {
                 this.channelDeferred?.reject(
                     CreateProcessingError(error, "realizeFluidDataStoreContext", undefined /* message */));
-            });
+                this.logger.sendErrorEvent(
```

Comment:
Couple things:
1. Let's log the same error we reject with
2. Should `this.id` be tagged as `packageData`? (Sorry I haven't renamed it yet btw)

Reply:
I do not like # 1, as that may result in loss of information.
I'd rather see raw data, and developers can follow code to understand how data is transformed downstream

Reply:
Look at what `prepareErrorObject` does and what `wrapError` does.  I think the only info preserved in either case is preserved in both:  `message` and `stack`.  I'll check for myself in a few min to make sure I'm right :)

Reply:
It does not matter what it preserves and what it does not (today). I prefer (where it's possible and does not make analyzes harder) to preserve original errors, as much as possible.
That said, if we want to bucket these errors together with other errors of similar type ("processing errors"), then yes, we might want to log wrapped error.

Reply:
Ok, I changed it :)